communication/marketing-team/README.md

```diff
@@ -1,57 +1,41 @@
-Future home for all things marketing team for contributor experience
-
-# Charter
-view our charter in this [marketing folder] for more about us and how to join
+The Contributor Marketing Team is part of the community-management subproject in
+[Contributor Experience] view our [charter] to learn more about us
 
 ## Purpose  
 to better inform the Kubernetes contributor community and highlight their work to
 the broader ecosystem
 
-# Core Marketing Team roles
+# Core Marketing Team
 
 | Role | Handbook | Notes |
```

Comment:
Given the lead/shadow model, I think shifting the `Lead` specification to the 3rd column would make the roles and titles a bit clearer, so:

Reply:
I made some further modifications because I also thought it was odd originally but with this comment and @jonasrosland, I had an idea of adding an 'area' section which makes it a bit clearer with the addition of 'team' and 'lead' names. wdyt?